git_machete/cmd.py

```diff
@@ -2090,10 +2151,11 @@ def print_new_line(new_status):
         cb = current_branch()
         expect_in_managed_branches(cb)
 
+    b: str
     for b in itertools.dropwhile(lambda x: x != cb, managed_branches):
```

Comment:
Whoops no syntax for declaring iterator variable type inside of `for`?... like `for b: str in ...`?

Reply:
unfortunately :c that's the only way